sickbeard/metadata/mede8er.py

```diff
@@ -373,19 +383,23 @@ def write_show_file(self, show_obj):
 
         try:
             if not ek(os.path.isdir, nfo_file_dir):
-                logger.log(u"Metadata dir didn't exist, creating it at " + nfo_file_dir, logger.DEBUG)
+                logger.log(u'Metadata directory did not exist, creating it at {path}'.format
+                           (path=nfo_file_dir), logger.DEBUG)
                 ek(os.makedirs, nfo_file_dir)
                 helpers.chmodAsParent(nfo_file_dir)
 
-            logger.log(u"Writing show nfo file to " + nfo_file_path, logger.DEBUG)
+            logger.log(u'Writing show nfo file to {path}'.format
+                       (path=nfo_file_path), logger.DEBUG)
 
             nfo_file = io.open(nfo_file_path, 'wb')
 
-            data.write(nfo_file, encoding="utf-8", xml_declaration=True)
+            data.write(nfo_file, encoding='utf-8', xml_declaration=True)
             nfo_file.close()
             helpers.chmodAsParent(nfo_file_path)
         except IOError as e:
-            logger.log(u"Unable to write file to " + nfo_file_path + " - are you sure the folder is writable? " + ex(e),
+            logger.log(u'Unable to write file to {path} - '
+                       u'are you sure the folder is writable? {exception}'.format
```

Comment:
You know, until now I had never looked at the purpose of `ex`?  With unicode formatted logging there's absolutely no need for it.